src/Umbraco.Core/Security/BackOfficeUserManager.cs

```diff
@@ -507,15 +507,22 @@ private static string NewSecurityStamp()
 
         #endregion
 
-        public override Task<IdentityResult> SetLockoutEndDateAsync(int userId, DateTimeOffset lockoutEnd)
+        public override async Task<IdentityResult> SetLockoutEndDateAsync(int userId, DateTimeOffset lockoutEnd)
```

Comment:
Unfortunately this is a public method in a public class. Changing the method is a breaking changes as it will require everyone using it to have to recompile the code.
So we'll have to keep this one non-async for now!

Reply:
We can change to async, this isn't changing the method signature at all. It's already async enabled since its returning a Task. Same as the other method. All the async keyword does is modify the IL used to call the method.